### DISCUSSION TITLE
Is there an cloup format option to limit or format a long click.Choice option list?

### DISCUSSION THREAD

**Discussion by hiwixxl** [janluke/cloup#86] [Q&A] [answered]
I have a click.Choice option with show_default=True within a cloup script, which runs to the full width of the terminal because of 25 possible choices. It's nearly unreadable. It would help, if there's a setting for this case. I tried some settings, but even "max_width=100" did not help. Click shows the same effect.

**Comment by janluke** [comment]
No, there's not. You may consider opening this issue in the Click repository.

**Comment by janluke** [comment, marked as answer]
Actually, you can use the `metavar` argument of option to override that.
